theme/js/components/search/search.vue

```diff
@@ -1,184 +1,192 @@
-      @click="extendedForm = !extendedForm"
-    ></div>
-  </div>
-  <div class="row-inline fr-mt-3v justify-between align-items-center">
-    <h1 class="fr-m-0 fr-h4">
-      {{ $t("Datasets") }}<sup>{{ totalResults }}</sup>
-    </h1>
-    <a :href="reuseUrl" title="" class="nav-link fr-text--sm fr-mb-0 fr-displayed-md fr-mt-3v">
-      {{ $t("Search reuses") }}
-    </a>
-  </div>
-  <section class="search-filters fr-p-2w fr-p-md-0" :class="{ active: extendedForm }">
-    <h2 class="fr-mt-md-2w fr-mb-2w fr-mb-md-1w fr-text--sm">
-      {{ $t("Search filters") }}
-    </h2>
-    <div class="filters-wrapper fr-p-md-3v">
-      <div class="fr-grid-row fr-grid-row--gutters justify-between align-items-center">
-        <div class="fr-col-12 fr-col-md-6 fr-col-lg-3">
-          <Suggestor
-            :placeholder="$t('Organizations')"
-            :searchPlaceholder="$t('Search an organization...')"
-            listUrl="/organizations/?sort=-followers"
-            suggestUrl="/organizations/suggest/"
-            entityUrl="/organizations/"
-            :values="facets.organization"
-            :onChange="handleSuggestorChange('organization')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-6 fr-col-lg-3">
-          <Suggestor
-            :placeholder="$t('Tags')"
-            :searchPlaceholder="$t('Search a tag...')"
-            suggestUrl="/tags/suggest/"
-            :values="facets.tag"
-            :onChange="handleSuggestorChange('tag')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-5 fr-col-lg-3">
-          <Suggestor
-            :placeholder="$t('Licenses')"
-            :searchPlaceholder="$t('Search a license...')"
-            listUrl="/datasets/licenses/"
-            :values="facets.license"
-            :onChange="handleSuggestorChange('license')"
-          />
-        </div>
-        <div class="fr-col-12 fr-col-md-5 fr-col-lg-2">
-          <Suggestor
-            :placeholder="$t('Formats')"
-            :searchPlaceholder="$t('Search a format...')"
-            suggestUrl="/datasets/suggest/formats/"
-            :values="facets.format"
-            :onChange="handleSuggestorChange('format')"
-          />
-        </div>
-        <div
-          class="fr-col-2 fr-col-lg-1 fr-displayed-md text-align-center text-align-right-lg"
-        >
-          <button
-            class="fr-btn fr-btn--secondary fr-btn--secondary-grey-200 text-grey-380 fr-px-2w"
-            @click="extendedForm = !extendedForm"
-            data-cy="extend-form"
-          >
-            <span v-if="!extendedForm">…</span>
-            <span v-else>X</span>
-          </button>
-        </div>
+  <form class="fr-pt-3v" @submit.prevent="search">
+    <div class="fr-grid-row fr-grid-row--middle justify-between search-bar" ref="searchRef">
+      <SearchInput
+        :onChange="handleSearchChange"
+        :value="queryString"
+        :placeholder="$t('Search for data...')"
```

Comment:
@agarrone Could we add a wording to make the total number of datasets appear here?
Ex: Search among 45000 datasets / Rechercher parmis 45000 jeux de données.
What do you think?

Reply:
The current placeholder is not the one on Figma, which is "Ex. Résultats élections 2022"

Reply:
We could add the total number of objects on `Search a dataset on uData` title instead of the placeholder?

Reply:
Let's show this information on the title using `current_site.metrics['datasets']` value :)